README.md

```diff
@@ -161,6 +161,15 @@ You can see the [whole fix as a commit message on GitHub](https://github.com/r-l
 See also our blog post at
 https://www.tidyverse.org/articles/2019/05/resource-cleanup-in-c-and-the-r-api/
 
+It important to keep in mind that when the cleanup functions run, the
```

Comment:
This explanation feels a bit obscure. Maybe something along these lines?

> Note that the cleanup functions cannot generally assume that stack-allocated data are still around at the time they are called. This is usually not a problem since cleanup is mostly about objects allocated on the heap with non-automatic storage. If needed, you can protect stack-allocated data from being unwound by using `r_with_cleanup_context()`. This becomes the point at which cleanup functions are called, which ensures any object allocated on the stack before that point are still around.

Reply:
ok